spec/services/bulkrax/bulkrax_configuration_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# This spec is to ensure that configuration removed from the Engine is still being read as it was before the change
+RSpec.describe Bulkrax do
```

Comment:
Seems a bit unnecessary to test what is essentially a configuration.  Can't the Bulkrax config go in an initializer not a service?